106/CatalogHW/CatalogHW/Model/Playlist.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Playlist.swift
+//  CatalogHW
+//
+//  Created by Полина Рыфтина on 19.10.2022.
+//
+
+struct Song {
+    let songTitle: String
+    let author: String
+    let duration: Double
+}
+
+struct Playlist {
+    let playlistTitle: String
+    let numberOfTracks: Int
+    let totalDuration: Double
+    let isDownloaded: Bool
+    let allSongs: [Song]
+    
+    public static func randomPlaylist() -> Playlist {
+        let randomSongs = randomSongs(countOfSongs: Int.random(in: 3...9))
+        let randomPlaylist = Playlist(
+            playlistTitle: randomPlaylistTitle(),
+            numberOfTracks: randomSongs.count,
+            totalDuration: totalDuration(songs: randomSongs),
+            isDownloaded: Bool.random(),
+            allSongs: randomSongs)
+        return randomPlaylist
+    }
+    
+    private static func totalDuration(songs: [Song]) -> Double {
+        var totalDuration: Double = 0
+        for song in songs {
+            totalDuration += song.duration
+        }
+        totalDuration = round(totalDuration * 100)/100
```

Comment:
Не знаю, как у тебя оно билдилось, подойти на паре, посмотрим. пока за это минус
```suggestion
        totalDuration = ((totalDuration * 100)/100).rounded()
```